roles/cdi/tasks/deprovision.yml

```diff
@@ -1,46 +1,102 @@
 ---
 - name: Render {{ cdi_namespace }} ResourceQuota deprovision yaml
   template:
-    src:  cdi-resourcequota.yml
+    src: cdi-resourcequota.yml.j2
     dest: /tmp/cdi-deprovision-resourcequota.yml
 
 - name: Delete {{ cdi_namespace }} ResourceQuota
   command: kubectl delete -f /tmp/cdi-deprovision-resourcequota.yml -n {{ cdi_namespace }} --ignore-not-found
 
-- name: Check that cdi-controller.yaml still exists in /tmp
+- name: Check that cdi-operator-cr.yaml still exists in /tmp
   stat:
-    path: "/tmp/cdi-controller.yaml"
-  register: cdi_template
+    path: "/tmp/cdi-operator-cr.yaml"
+  register: cdi-operator-cr
 
-- name: Get cdi-controller.yaml
-  block:
-    - name: Check for cdi-controller.yaml.j2 template in {{ cdi_template_dir }}
-      stat:
-        path: "{{ cdi_template_dir }}/cdi-controller.yaml.j2"
-      register: byo_template
+- name: Render CDI operator cr
+  template:
+    src: "cdi-operator-cr.yaml.j2"
```

Comment:
Can't use a '-' for variables in ansible; only '_'.  Change to ```cdi_operator_cr.yaml.j2```

Reply:
it should be just for variable name, not for the content of the variable

Reply:
right, just for the name.